google-fcm/src/main/scala/akka/stream/alpakka/google/firebase/fcm/javadsl/GoogleFcm.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.google.firebase.fcm.javadsl
+
+import akka.NotUsed
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.japi.Pair
+import akka.stream.alpakka.google.firebase.fcm.FcmFlowModels.{FcmFlowConfig, FcmResponse}
+import akka.stream.alpakka.google.firebase.fcm.impl.FcmSender
+import akka.stream.alpakka.google.firebase.fcm.{FcmFlowModels, FcmNotification}
+import akka.stream.scaladsl.{Flow, Sink}
+import akka.stream.{javadsl, Materializer}
+
+object GoogleFcm {
```

Comment:
Most other connectors have the factories split into Source, Flow, Sink.
Might be an idea to adhere to that scheme and make it `GoogleFcmFlow` and `GoogleFcmSink`.